parm/nems.configure.med_atm_ocn_ice_wav.IN

```diff
@@ -64,14 +65,17 @@ runSeq::
   @@[coupling_interval_slow_sec]
     MED MedPhase_prep_ocn
     MED -> OCN :remapMethod=redist
+    OCN -> WAV 
+    WAV -> OCN :srcMaskValues=1
     OCN
     @@[coupling_interval_fast_sec]
       MED MedPhase_prep_ice
       MED MedPhase_prep_atm
       MED -> ATM :remapMethod=redist
       MED -> ICE :remapMethod=redist
-      WAV -> ATM
-      ATM -> WAV 
+      WAV -> ATM :srcMaskValues=1
```

Comment:
I am curious what value of Z0 does wave have at the first time step in this run sequence?

Reply:
If WW3 has an IC, on the first call to export this variable in the NUOPC cap, it calls the physics routines to calculate the roughness see: https://github.com/NOAA-EMC/WW3/blob/develop/model/ftn/wmesmfmd.ftn#L5605-L5622

Reply:
Thanks.